### ISSUE TITLE
How does one persistently enable segments?

### ISSUE THREAD

**Issue by Apromixately** [TheBB/spaceline#205]
It seems calling spaceline-toggle-org-clock-on from user-config in .spacemacs works but it would be good to mention the intended way in the documentation.

**Comment by TheBB** [comment]
Yep, that is the intended way. Either that or set the `spaceline-org-clock-p` variable. I'll leave
this open for documentation
